StripePayments/StripePayments/Source/PaymentHandler/STPPaymentHandler.swift

```diff
@@ -2177,6 +2179,17 @@ public class STPPaymentHandler: NSObject {
         }
         return STPPaymentHandlerError(code: errorCode, loggingSafeUserInfo: userInfo) as NSError
     }
+
+    func callContextWillDismissIfNeeded(_ context: (any STPAuthenticationContext)?, _ viewController: UIViewController?) {
```

Comment:
Took me a while to wrap my head around the naming of this function, but if i'm correct -it literally executing context.willDismiss.. but.. seems to execute `authenticationContextDidDismiss()`

Reply:
It's just a typo, thanks for catching.